src/offset/local/tz_info/rule.rs

```diff
@@ -228,123 +230,32 @@ impl AlternateTime {
         }
     }
 
-    fn find_local_time_type_from_local(
+    fn find_local_offset_from_local(
```

Comment:
Can this function also have a 1:1 test case? `fn test_find_local_offset_from_local()` with many asserts?
I'd have more confidence if it did.

Reply:
This function is covered by the same test as `TzInfo::lookup_with_dst_transitions`